src/grimoirelab/core/config/settings.py

```diff
@@ -273,9 +273,7 @@
 # https://github.com/rq/django-rq
 #
 
-Q_PERCEVAL_JOBS = os.environ.get('GRIMOIRELAB_Q_PERCEVAL_JOBS', 'default')
-Q_STORAGE_ITEMS = os.environ.get('GRIMOIRELAB_Q_STORAGE_ITEMS', 'items')
-Q_EVENTS = os.environ.get('GRIMOIRELAB_Q_EVENTS', 'events')
+Q_EVENTIZER_JOBS = os.environ.get('GRIMOIRELAB_Q_EVENTIZER_JOBS', 'default')
```

Comment:
I think we can use the prefix `GRIMOIRELAB_` to all these variables that can be configured to make it clear they are part of this app and don't come from other.